ui/components/MesherySettings.js

```diff
@@ -10,14 +10,13 @@ import Typography from '@material-ui/core/Typography';
 import { AppBar, Paper, Tooltip, IconButton } from '@material-ui/core';
 import CloseIcon from "@material-ui/icons/Close";
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faArrowLeft, faCloud, faPoll, faTachometerAlt } from '@fortawesome/free-solid-svg-icons';
+import { faArrowLeft, faCloud, faPoll } from '@fortawesome/free-solid-svg-icons';
 import { faMendeley } from '@fortawesome/free-brands-svg-icons';
 import Link from 'next/link';
 import MeshConfigComponent from './MeshConfigComponent';
 import GrafanaComponent from './GrafanaComponent';
 import MeshAdapterConfigComponent from './MeshAdapterConfigComponent';
 import PrometheusComponent from './PrometheusComponent';
-import MesherySettingsPerformanceComponent from "../components/MesherySettingsPerformanceComponent";
```

Comment:
@sudo-NithishKarthik I believe that we're going to want to retain this component on the /settings page, but have it invoke a different API endpoint than it has in the past.

Reply:
Okay, so we will have it in both places ?